core/domain/feedback_services.py

```diff
@@ -171,12 +171,18 @@ def create_message(
             exploration_id = thread.exploration_id
         if message_id == 0:
             # New thread.
-            event_services.FeedbackThreadCreatedEventHandler.record(
-                exploration_id)
+            if not (
```

Comment:
Add a test that ensures that other threads will not trigger this, but exploration threads will.

In general if you make an error then that indicates a hole in the test coverage. So you need to write a test that would have caught the error.

Reply:
Done